woo-razorpay.php

```diff
@@ -491,9 +498,7 @@ protected function createRazorpayOrderId($orderId, $sessionKey)
             }
             catch (Exception $e)
             {
-                $message = $e->getMessage();
-
-                return 'RAZORPAY ERROR: Order creation failed with the message \'' . $message . '\'';
+                return $e;
```

Comment:
This will catch the exception and return it. Won't fix the problem of the exception being caught.

Reply:
it's getting caught in parent calling "getRazorpayPaymentParams" function.

Reply:
That will also work if you just call `$api->order->create($data)` directly. What is the need for the try-catch and return?

Reply:
It's written to handle the error in proper way and display the message properly.